extensions/scrapbox/src/components/Pages.tsx

```diff
@@ -0,0 +1,20 @@
+import { ActionPanel, List, Action } from "@raycast/api";
+
+export function Pages(props: { pages: [] }) {
+  return (
+    <List>
+      {props.pages?.map((page: any) => (
+        <List.Item
+          key={page.id}
+          icon="list-icon.png"
+          title={page.title}
+          actions={
+            <ActionPanel>
+              <Action.OpenInBrowser url={encodeURI(`https://scrapbox.io/zakuni/${page.title}`)}></Action.OpenInBrowser>
```

Comment:
Is this hardcoded to your scrapbox?

Reply:
Oops, yes, I've mistakenly left a hardcoded link to my scrapbook. I'm going to fix it.
Thanks for your review!

Reply:
@pernielsentikaer Hi, actually `Pages.tsx` itself was the old unused file I'd forgotten to remove. I've already deleted the file and pushed the branch. 
Would you please review again? Thanks.